ginkgo-kubernetes-all.Jenkinsfile

```diff
@@ -73,9 +73,49 @@ pipeline {
                 TESTDIR="${WORKSPACE}/${PROJ_PATH}/test"
             }
 
-            steps {
-                sh 'cd ${TESTDIR}; K8S_VERSION=1.11 vagrant up --no-provision'
-                sh 'cd ${TESTDIR}; K8S_VERSION=1.12 vagrant up --no-provision'
+            parallel {
+                stage('Boot vms 1.11') {
+                    environment {
+                        TESTED_SUITE="1.11"
+                        GOPATH="${WORKSPACE}/${TESTED_SUITE}-gopath"
+                        TESTDIR="${GOPATH}/${PROJ_PATH}/test"
+                    }
+                    steps {
+                        sh 'mkdir -p ${GOPATH}/src/github.com/cilium'
+                        sh 'cp -a ${WORKSPACE}/${PROJ_PATH} ${GOPATH}/${PROJ_PATH}'
```

Comment:
I'm ok with this, but I had problems with something like that in the past, some files are out-of-sync, so maybe a rm of folder will be good thing to add.

Reply:
This has been running in main jenkinsfile in that form for over two weeks without problems now. We also have much quicker node turnaround, which I think also helps with workspaces being clean. Can we add the `rm` if we see this happening again?